api/proto/canary/aws.proto

```diff
@@ -40,14 +40,17 @@ message AwsAccount {
   // `~/.aws/credentials`. Defaults to `default`.
   string profileName = 5;
 
-  // The endpoint used to reach the service implementing the AWS API. Typical
-  // use is with Minio.
+  // The endpoint used to reach the service implementing the S3 API. Typically
+  // you would use this with an S3 clone, like Minio.
   string endpoint = 6;
 
   // The default access key used to communicate with AWS.
   string accessKeyId = 7;
 
-  // If enabling S3, include CONFIGURATION_STORE and/or OBJECT_STORE in this
-  // list.
+  // Use METRICS_STORE for AWS monitoring.
```

Comment:
This might have been correct as-is; not actually sure we support an AWS metrics store based on our [docs](https://spinnaker.io/setup/canary/#identify-your-metrics-provider)?

Reply:
But they'd want to include `METRICS_STORE`, regardless what metrics provider they use, right?

Reply:
I don't think so, I think you'd only want to specify `METRICS_STORE` for accounts for integrations that could be used to supply metrics. I think the AWS integration is only for S3 as a persistent store, for either (or both) of `CONFIGURATION_STORE` and `OBJECT_STORE`.

Reply:
https://github.com/spinnaker/kayenta/blob/master/kayenta-web/config/kayenta.yml#L57

Reply:
Only has object store and configuration store listed.

Reply:
:+1: 
Thanks, Maggie and Justin!